Lib/test/test_os.py

```diff
@@ -4349,6 +4349,8 @@ def test_timerfd_non_blocking(self):
         # confirm if timerfd is readable and read() returns 1 as bytes.
         self.assertEqual(self.read_count_signaled(fd), 1)
 
+    @unittest.skipIf(sys.platform.startswith('netbsd'),
+                     "gh-131263: Skipp on NetBSD due to system freeze with negative timer values")
```

Comment:
Can you please truncate to 80 columns? Also, there is a typo:

```suggestion
                     "gh-131263: Skip on NetBSD due to system freeze with negative timer values")
```